discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -341,49 +360,50 @@ def wait_for_pending_for_input_status(self):
             timeout=consts.PENDING_USER_ACTION_TIMEOUT
         )
 
-    def wait_for_at_least_one_host_to_boot_during_install(self, nodes_count=1):
+    def wait_for_at_least_one_host_to_boot_during_install(self):
         utils.wait_till_at_least_one_host_is_in_stage(
             client=self.api_client,
             cluster_id=self.id,
             stages=[consts.HostsProgressStages.REBOOTING],
-            nodes_count=nodes_count
+            nodes_count=self._config.nodes_count
```

Comment:
please set default 1 instead of self._config.nodes_count

Reply:
Why? This is a different logic..

Reply:
why it is different? it was nodes_count=1 before the change
method name also indicate that at least 1 node is required :) 
wait_for_**at_least_one_host_to**_boot_during_install